backend/app/modules/chat/llm_processing.py

```
@@ -53,13 +53,16 @@ def ask_llm(question, docs):
             {"role": "system", "content": "Use only the context to answer."},
             {"role": "user", "content": prompt},
-        ],
-    )
-    logger.info("LLM response retrieved successfully.")
-    return response.choices[0].message.content
+        ],  
+        )
+        logger.info("LLM response retrieved successfully.")
+        return response.choices[0].message.content
+    except Exception as e:
+        logger.error(f"Error during LLM interaction: {e}")
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)